pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -261,7 +261,14 @@ func (r *rbac) getNamespacesToBeReconciled(ctx context.Context) ([]corev1.Namesp
 		// We must make sure that the default SCC is in force via the ClusterRole.
 		sccRoleBinding, err := r.kubeClientSet.RbacV1().RoleBindings(ns.Name).Get(ctx, pipelinesSCCRoleBinding, metav1.GetOptions{})
 		if err != nil {
-			return nil, err
+			// https://github.com/tektoncd/operator/issues/2100
```

Comment:
@jkhelil we may remove this GH issue reference
(AFAIK, we do not have incomplete work from this issue mentioned here.)